client/src/pages/AssignRoles.js

```diff
@@ -0,0 +1,22 @@
+/*
+UI for assigning roles.
+*/
+
+import React from "react";
+import { Button } from '@material-ui/core';
+import "../styles/assignRoles.css";
+import AvailableRoles from '../components/AvailableRoles';
+
+export default function AssignRoles() {
+
+    return (
+        <div className="container">
+            <h1>New Game</h1>
+            <hr />
+            <h2>Available Roles</h2>
+            <AvailableRoles />
+            <br />
```

Comment:
It's better practice to use styling rather than a br tag where possible

Reply:
Used Styling and removed <br />.